README.md

```diff
@@ -219,3 +220,75 @@ Cols: []
 -----------------------------------------
 Excel files xxx.xlsx and yyy.xlsx match
 </pre>
+
+## Unified Diff output format
+* Diffs are reported in the "unified diff" style, with no surrounding context (_i.e._, a la `diff -U0`).
+* Each sheet containing a diff or an extra cell begins with a header as follows:
+		--<FileName1>!<SheetName>
+		++<FileName2>!<SheetName>
+* Each row containing a diff or an extra cell begins with a line that identifies the specific cell range as follows:
+		@@ <Cell1>,<CellN> <Cell1>,<CellN>  @@
+* Each diff or extra cell is reported as follows:
+		-<Cell1Value1>
+		-...
+		-<Cell2Value1>
+		-<CellnValue1>
+		+<Cell1Value2>
+		-...
+		+<Cell2Value2>
+		+<CellnValue2>
```

Comment:
I considered a row-by-row diff.  You had started with a cell-by-cell diff, and I tried to stay with that concept.  And honestly, if that's the desire, I probably would probably just export the worksheets to CSV and use the standard `diff` against those files.  That wouldn't work for more complex spreadsheets, but building a script with something like Gnumeric's `ssconvert` might be pretty straightforward.

What would be really awesome would be to use a diff engine that could recognize things like the insertion of a row and treat it that way, instead of as a change to every cell below that row.  And likewise a column.  But that's an idea for another day :-)

I appreciate this tool.  I used it to compare some very complicated spreadsheets and was thrilled with the results.  As a programmer, I lean kind of naturally toward `diff -u`, and since you had a TODO on it, it seemed logical to follow through.  Once I did that, it made my comparisons even easier to understand.

Reply:
Oh ya - recognizing insertion / deletion of row / column will be totally sweet. That's the first line in my TODO: Shifting cells, columns, rows

I created this tool way back (more than 10 years ago) when our product was outputting Excel files and I wanted to write some automated tests. Don't work much with spreadsheets anymore.

The other thoughts about this tool top of mind are - google sheets as input, visual diff